# Análisis de Datos de Seguros - Enfoque Step-by-Step y Caso de Negocio

Este notebook está diseñado para que resuelvas problemas paso a paso utilizando Python (usando funciones básicas como .mean(), filtros, value_counts y groupby) y que puedas comparar los resultados con el objetivo propuesto. Además, se plantea un caso de negocio ficticio en el que se asigna un gasto en marketing (CAC) para evaluar la rentabilidad y tomar decisiones estratégicas.

El objetivo es evaluar tu habilidad para:
- Explorar y analizar datos en Python.
- Resolver problemas de análisis paso a paso.
- Comparar resultados con objetivos estratégicos.
- Formular conclusiones y recomendaciones basadas en los datos.

¡Comencemos!

In [2]:
!wget https://github.com/javierherrera1996/IntroMarketingAnalytics/raw/refs/heads/main/PrimerCorte/WA_Fn-UseC_-Marketing-Customer-Value-Analysis.csv.zip
!unzip WA_Fn-UseC_-Marketing-Customer-Value-Analysis.csv.zip

--2025-03-12 23:06:40--  https://github.com/javierherrera1996/IntroMarketingAnalytics/raw/refs/heads/main/PrimerCorte/WA_Fn-UseC_-Marketing-Customer-Value-Analysis.csv.zip
Resolving github.com (github.com)... 140.82.113.4
Connecting to github.com (github.com)|140.82.113.4|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://raw.githubusercontent.com/javierherrera1996/IntroMarketingAnalytics/refs/heads/main/PrimerCorte/WA_Fn-UseC_-Marketing-Customer-Value-Analysis.csv.zip [following]
--2025-03-12 23:06:40--  https://raw.githubusercontent.com/javierherrera1996/IntroMarketingAnalytics/refs/heads/main/PrimerCorte/WA_Fn-UseC_-Marketing-Customer-Value-Analysis.csv.zip
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 352796 (345K) [

## Paso 1: Exploración Inicial del Dataset
1. Carga el dataset `WA_Fn-UseC_-Marketing-Customer-Value-Analysis.csv`.
2. Revisa las primeras filas, dimensiones y la información general del dataset.

_Observa la estructura de los datos: ¿Qué columnas tienes? ¿Se nota algún dato atípico?_

In [3]:
# Cargar el dataset y ver primeras filas
import pandas as pd
import numpy as np

df = pd.read_csv('WA_Fn-UseC_-Marketing-Customer-Value-Analysis.csv')
pd.set_option('display.max_columns', None)


## Paso 2: Análisis del Customer Lifetime Value (CLV)
1. Calcula el CLV promedio y su desviación estándar.
2. Reflexiona: ¿Qué indica una alta desviación? ¿Podrían existir segmentos de clientes con comportamientos muy distintos?

In [15]:
clv_mean = df["Customer Lifetime Value"].mean()
clv_std = df["Customer Lifetime Value"].std()
print(f"El CLV promedio es {clv_mean} y la desviación estándar es {clv_std}")

El CLV promedio es 8004.940474987081 y la desviación estándar es 6870.967608356924


Una alta desviaciòn indica que los datos obtenidos en la base de datos en cuanto al CLV varian unos de otros en cuanto al promedio. Por consiguiente, si podrìa existir segmentos de clientes con comportamientos muy distintos y esto a su vez podrìa ser resultado de clientes con habitos y necesidades diferentes.

## Paso 3: Evaluación de la Columna Response
1. Usa `value_counts()` para ver cuántos clientes respondieron "Yes" y "No".
2. Calcula el porcentaje de respuestas "Yes".

Pregunta: ¿La proporción de "Yes" es baja? ¿Qué podría indicar esto respecto a la estrategia de respuesta?

In [21]:
df['Response'].value_counts()

Response
No     7826
Yes    1308
Name: count, dtype: int64

In [ ]:
df['Response'].value_counts()
response_counts = df['Response'].value_counts()
(response_counts['Yes'] / response_counts.sum()) * 100

14.320122618786948

Al ser el porcentaje de respuesta "Yes" es de alguna manera bajo, esto podría indicar que una pequeña proporción de clientes de la compañìa estàn respondiendo de manera positiva ante la propuesta o inicitiva que se este presentando, lo que podrìa hacer evidente falencias en el mensaje que se le esta siendo enviado a los clientes.
Esto podrìa inidica en cuanto a la estrategia de respuesta que no se siendo lo suficientemente atractivo para los clientes por lo que podrìa terminar siendo irrelevante, como al mismo tiempo, significaria que la empresa pudo llevar a cabo una mala segmentaciòn de sus clientes por lo que no estarìan conectado con la oferta de la compañìa.

## Paso 4: Análisis de la Prima Mensual (Monthly Premium Auto) por Coverage
1. Agrupa los datos por `Coverage` y calcula la media de `Monthly Premium Auto`.
2. Compara los promedios obtenidos para cada tipo de Coverage.

Pregunta: ¿Existen diferencias notables entre las coberturas? ¿Qué implicaciones tendría esto en la estrategia de precios?

In [25]:
df.groupby('Coverage')['Monthly Premium Auto'].mean()



Coverage
Basic        82.173851
Extended    103.579504
Premium     133.381068
Name: Monthly Premium Auto, dtype: float64

In [32]:
Monthly_Premium_Auto = df.groupby("Coverage")["Monthly Premium Auto"].mean()
max_coverage = Monthly_Premium_Auto.idxmax()
min_coverage = Monthly_Premium_Auto.idxmin()
diff = Monthly_Premium_Auto.max() - Monthly_Premium_Auto.min()

print(f"\nLa cobertura con la prima mensual promedio más alta es: {max_coverage} "
      f"(${Monthly_Premium_Auto[max_coverage]:.2f})")
print(f"La cobertura con la prima mensual promedio más baja es: {min_coverage} "
      f"(${Monthly_Premium_Auto[min_coverage]:.2f})")
print(f"Diferencia entre la cobertura más cara y la más barata: ${diff:.2f}")


La cobertura con la prima mensual promedio más alta es: Premium ($133.38)
La cobertura con la prima mensual promedio más baja es: Basic ($82.17)
Diferencia entre la cobertura más cara y la más barata: $51.21


Si se encuentran diferencias significativas dentro de las coberturas implicaria que las coberturas conmayor beneficio cuentan con una prima mucho màs alta, lo cual dentro de la estrategia de precios puede ser aprovechado implementando un precio màs alto dado todos los beneficios aadicionales que traen.

## Paso 5: Análisis del Ingreso (Income) según EmploymentStatus
1. Agrupa los datos por `EmploymentStatus` y calcula el ingreso promedio.

Pregunta: ¿Qué diferencias encuentras entre los grupos? ¿Cómo podría influir esto en la segmentación de clientes?

In [27]:
df.groupby('EmploymentStatus')['Income'].mean()

EmploymentStatus
Disabled         20045.582716
Employed         56384.884521
Medical Leave    20292.770833
Retired          20554.960993
Unemployed           0.000000
Name: Income, dtype: float64

Las diferencias que se encuentra en el grupo es que su nivel de ingreso varia de manera considerable dependiendo con el estado de empleabilidad, como se puede observar que los que se encuentran empleados tienen un nivel de ingresos muchisimo mayor al de los que se encuentran sin un empleo. Esto puede influir a la segmentacion de clientes dado que esto permite definir la estrategia a desarollar ya sea manejar precios bajos partiendo como clientes meta a los trabajadores de medio tiempo y a los que tengan un ingreso menos asì como tambien se podrìan ofrecer productos premium enfocados en los clientes que se encuentran empleados.

## Paso 6: Análisis del CLV por Estado
1. Agrupa los datos por `State` y calcula el CLV promedio.

Pregunta: ¿Qué estado muestra el mayor CLV? ¿Qué factores regionales podrían explicar esto?

In [34]:
df.groupby('State')['Customer Lifetime Value'].mean()

State
Arizona       7861.341489
California    8003.647758
Nevada        8056.706839
Oregon        8077.901191
Washington    8021.472273
Name: Customer Lifetime Value, dtype: float64

El estado que muestra un mayor CLV es Oregon, algunos de los factores regionales que podrìan explicar esto son los niveles de ingreso de la regiòn, lo que significarià mayores ingresos disponibles para gastar por lo que tendrìan primas màs altas. Por otro lado, las tendencias demograficas.

## Paso 7: Análisis del Total Claim Amount según el Número de Pólizas
1. Separa a los clientes con más de 1 póliza de aquellos con 1 sola póliza.
2. Calcula el Total Claim Amount promedio para cada grupo.

Pregunta: ¿Qué diferencias encuentras y qué podrían indicar respecto al riesgo y comportamiento de reclamaciones?

In [42]:
df[df['Number of Policies'] == 1]

Customer       State  Customer Lifetime Value Response  Coverage  \
0     BU79786  Washington              2763.519279       No     Basic   
4     HB64268  Washington              2813.692575       No     Basic   
16    FV94802      Nevada              2566.867823       No     Basic   
17    OE15005  California              3945.241604       No     Basic   
20    ZK25313      Oregon              2872.051273       No     Basic   
...       ...         ...                      ...      ...       ...   
9120  RN61682  California              2114.738469       No     Basic   
9121  BB10681  California              4140.648654       No  Extended   
9128  YM19146  California              4100.398533       No   Premium   
9130  PK87824  California              3096.511217      Yes  Extended   
9133  Y167826  California              2611.836866       No  Extended   

                 Education Effective To Date EmploymentStatus Gender  Income  \
0                 Bachelor           2/24/11         Employed      F   56274   
4                 Bachelor            2/3/11         Employed      M   43836   
16    High School or Below            2/6/11    Medical Leave      M   25049   
17                 College            1/5/11    Medical Leave      M   28855   
20    High School or Below           2/19/11         Employed      M   57749   
...                    ...               ...              ...    ...     ...   
9120              Bachelor           2/19/11       Unemployed      F       0   
9121               College           1/17/11       Unemployed      F       0   
9128               College            1/6/11         Employed      F   47761   
9130               College           2/12/11         Employed      F   21604   
9133               College           2/14/11       Unemployed      M       0   

     Location Code Marital Status  Monthly Premium Auto  \
0         Suburban        Married                    69   
4            Rural         Single                    73   
16        Suburban        Married                    67   
17        Suburban        Married                   101   
20        Suburban         Single                    74   
...            ...            ...                   ...   
9120      Suburban         Single                    72   
9121      Suburban       Divorced                   123   
9128      Suburban         Single                   104   
9130      Suburban       Divorced                    79   
9133      Suburban         Single                    77   

      Months Since Last Claim  Months Since Policy Inception  \
0                          32                              5   
4                          12                             44   
16                         14                              7   
17                         12                             59   
20                         31                             21   
...                       ...                            ...   
9120                       30                             69   
9121                        5                             35   
9128                       16                             58   
9130                       14                             28   
9133                        3                             90   

      Number of Open Complaints  Number of Policies     Policy Type  \
0                             0                   1  Corporate Auto   
4                             0                   1   Personal Auto   
16                            0                   1   Personal Auto   
17                            0                   1   Personal Auto   
20                            0                   1   Personal Auto   
...                         ...                 ...             ...   
9120                          5                   1   Personal Auto   
9121                          2                   1  Corporate Auto   
9128                          0                   

In [43]:
df[df['Number of Policies'] > 1]

Customer       State  Customer Lifetime Value Response  Coverage  \
1     QZ44356     Arizona              6979.535903       No  Extended   
2     AI49188      Nevada             12887.431650       No   Premium   
3     WW63253  California              7645.861827       No     Basic   
5     OC83172      Oregon              8256.297800      Yes     Basic   
6     XZ87318      Oregon              5380.898636      Yes     Basic   
...       ...         ...                      ...      ...       ...   
9126  AC13887  California              4628.995325       No     Basic   
9127  TF56202  California              5032.165498       No     Basic   
9129  LA72316  California             23405.987980       No     Basic   
9131  TD14365  California              8163.890428       No  Extended   
9132  UP19263  California              7524.442436       No  Extended   

     Education Effective To Date EmploymentStatus Gender  Income  \
1     Bachelor           1/31/11       Unemployed      F       0   
2     Bachelor           2/19/11         Employed      F   48767   
3     Bachelor           1/20/11       Unemployed      M       0   
5     Bachelor           1/25/11         Employed      F   62902   
6      College           2/24/11         Employed      F   55350   
...        ...               ...              ...    ...     ...   
9126  Bachelor            1/9/11       Unemployed      M       0   
9127   College           2/12/11         Employed      M   66367   
9129  Bachelor           2/10/11         Employed      M   71941   
9131  Bachelor            2/6/11       Unemployed      M       0   
9132   College            2/3/11         Employed      M   21941   

     Location Code Marital Status  Monthly Premium Auto  \
1         Suburban         Single                    94   
2         Suburban        Married                   108   
3         Suburban        Married                   106   
5            Rural        Married                    69   
6         Suburban        Married                    67   
...            ...            ...                   ...   
9126      Suburban         Single                    67   
9127      Suburban       Divorced                    64   
9129         Urban        Married                    73   
9131      Suburban         Single                    85   
9132      Suburban        Married                    96   

      Months Since Last Claim  Months Since Policy Inception  \
1                          13                             42   
2                          18                             38   
3                          18                             65   
5                          14                             94   
6                           0                             13   
...                       ...                            ...   
9126                       25                             21   
9127                        6                             48   
9129                       18                             89   
9131                        9                             37   
9132                       34                              3   

      Number of Open Complaints  Number of Policies     Policy Type  \
1                             0                   8   Personal Auto   
2                             0                   2   Personal Auto   
3                             0                   7  Corporate Auto   
5                             0                   2   Personal Auto   
6                             0                   9  Corporate Auto   
...                         ...                 ...             ...   
9126                          0                   4  Corporate Auto   
9127                          0                   3   Personal Auto   
9129                          0                   2   Personal Auto   
9131                          3                   2  Corporate Auto   
9132                          0                   3 

In [38]:
df[df['Number of Policies'] == 1]['Total Claim Amount'].mean()

440.2785319286374

In [39]:
df[df['Number of Policies'] > 1]['Total Claim Amount'].mean()

430.6682882804692

La principal diferencia que se puede evidenciar es que los clientes que cuentan con una sola poliza, tienen un total de reclamaciones mayor a aquellos que cuentan con màs de 1.

## Paso 8: Análisis del CLV según Vehicle Class
1. Agrupa los datos por `Vehicle Class` y calcula el CLV promedio para cada clase.

Pregunta: ¿Qué clase de vehículo se asocia a un mayor CLV y qué implicaciones podría tener esto en la estrategia de producto?

In [46]:
df.groupby('Vehicle Class')['Customer Lifetime Value'].mean()

Vehicle Class
Four-Door Car     6631.726607
Luxury Car       17053.348399
Luxury SUV       17122.999134
SUV              10443.511816
Sports Car       10750.989331
Two-Door Car      6671.030732
Name: Customer Lifetime Value, dtype: float64

La clase de vehiculo que se asocia a un mayor CLV es el Luxury SUV. Esto tendrìa implicaciones en cuanto a la estrategia del producto dado que estos clientes podrìan ser clientes fieles en el largo plazo, por lo que llevarìa a la empresa a centrarse en um segmento de mercado con un mayor rendimiento como lo son los propietarios de autos de alto valor.

## Paso 9: Análisis de Months Since Last Claim según Education
1. Agrupa los datos por `Education` y calcula el promedio de `Months Since Last Claim`.

Pregunta: ¿Existen diferencias notables entre los niveles educativos? ¿Qué podría indicar sobre la gestión de reclamos?

In [48]:
df.groupby('Education')['Months Since Last Claim'].mean()

Education
Bachelor                14.834789
College                 15.432301
Doctor                  15.260234
High School or Below    14.978642
Master                  15.199730
Name: Months Since Last Claim, dtype: float64

La diferencia es notable pero no en mayor medida. De esta manera, es podrìa indicar que las personas con niveles educativos màs altos realizan en promedio presentan menos reclamos. Mientras, las personas con un nivel educativo menor han presentado mayores cantidades de reclamos.
Adicionalmente, Aquellos con mayor educación tienden a ser conductores más cuidadosos y conocedores de sus pólizas, lo que permite ofrecerles primas más bajas y beneficios adicionales. Por otro lado, los clientes con menor educación pueden beneficiarse de programas educativos sobre seguridad vial y prevención de accidentes, así como de pólizas con mayor soporte. Al segmentar a los clientes según su nivel educativo, las aseguradoras pueden personalizar sus estrategias de gestión de reclamos, ofreciendo programas de fidelización a los clientes de bajo riesgo y proporcionando apoyo educativo a aquellos que lo necesiten, mejorando así la satisfacción del cliente y reduciendo la frecuencia de reclamos.

## Paso 10: Análisis del Income según Marital Status y Gender
1. Realiza una segmentación cruzada agrupando por `Marital Status` y `Gender` para calcular el ingreso promedio.

Pregunta: ¿Qué diferencias observas? ¿Cómo podrían influir estas diferencias en la estrategia de segmentación de clientes?

In [50]:
df.groupby(['Marital Status', 'Gender'])['Income'].mean()

Marital Status  Gender
Divorced        F         37913.709450
                M         43681.934848
Married         F         44276.994962
                M         43304.110361
Single          F         23573.758120
                M         22005.776407
Name: Income, dtype: float64

Las principales diferencias que se pueden evidenciar son que los hombres divorciadas tienen mayores ingresos, asì como las mujeres casadas y las mujeres solteras.
Esto puede influir en la estrategia de segmentaciòn de clientes por un lado, las personas casadas, al tener ingresos generalmente más altos, representan un mercado potencial para ofertas premium y paquetes con beneficios adicionales. Por otro lado, los solteros, con ingresos más modestos, se beneficiarían más de opciones económicas y descuentos. En cuanto al género, las mujeres divorciadas y solteras, que suelen tener mayores ingresos que sus contrapartes masculinas, responderían positivamente a campañas que enfaticen la estabilidad y la protección financiera. En contraste, los hombres solteros, con ingresos más bajos, se sentirían atraídos por alternativas de bajo costo y flexibilidad

## Paso 11: Distribución de la Columna Response
1. Usa `value_counts()` para mostrar cuántos clientes hay en cada grupo de Response.

Pregunta: ¿La distribución de respuestas sugiere que se debe ajustar la estrategia de comunicación?

In [51]:
df['Response'].value_counts()


Response
No     7826
Yes    1308
Name: count, dtype: int64

Si, dado que la distribuciòn de respuesta "Si" es inferior a su contraparte "No", lo que podrìa significar que no se està llegando de manera eficaz a los clientes.


## Paso 12: Análisis de Quejas Abiertas por Coverage
1. Agrupa los datos por `Coverage` y calcula el número promedio de quejas abiertas (`Number of Open Complaints`).

Pregunta: ¿Existen coberturas con un número de quejas significativamente mayor? ¿Qué acciones se podrían tomar?

In [52]:
df.groupby('Coverage')['Number of Open Complaints'].mean()

Coverage
Basic       0.384698
Extended    0.398979
Premium     0.333738
Name: Number of Open Complaints, dtype: float64

Observamos que el número de quejas abiertas es ligeramente más alto para la cobertura Extended (0.398979) y Basic (0.384698) en comparación con la Premium Coverage (0.333738). En cuanto a que acciones se podrìan tomar se podrìan revisar bien sus pólizas para ver qué está causando problemas y arreglarlo, ver qué reclamos se repiten y tomar medidas.

## Paso 13: Análisis de la Antigüedad de las Pólizas según EmploymentStatus
1. Agrupa por `EmploymentStatus` y calcula el promedio de `Months Since Policy Inception`.

Pregunta: ¿Existe alguna relación entre el estado laboral y la antigüedad de las pólizas? ¿Qué podría implicar esto en la estabilidad del cliente?

In [54]:
df.groupby('EmploymentStatus')['Months Since Policy Inception'].mean()

EmploymentStatus
Disabled         47.469136
Employed         47.800456
Medical Leave    48.141204
Retired          48.531915
Unemployed       48.747087
Name: Months Since Policy Inception, dtype: float64

La relación entre el estado laboral y la antigüedad de las pólizas revela patrones significativos para las aseguradoras. Los clientes en licencia médica, jubilados y desempleados tienden a mantener sus pólizas por períodos más prolongados, lo que sugiere una base estable para la empresa, aunque requiere una gestión cuidadosa ante posibles fluctuaciones económicas. Por otro lado, los empleados activos, si bien muestran una antigüedad de póliza ligeramente menor, representan una oportunidad para fidelización a través de productos personalizados y opciones de pago flexibles. En consecuencia, las aseguradoras deben adoptar estrategias diferenciadas, adaptando sus ofertas y servicios a las necesidades específicas de cada segmento laboral, con el fin de garantizar tanto la satisfacción del cliente como la estabilidad a largo plazo de la cartera de pólizas.

## Paso 14: Análisis del Total Claim Amount según Policy Type
1. Agrupa por `Policy Type` y calcula el Total Claim Amount promedio para cada tipo de póliza.

Pregunta: ¿Qué diferencias se observan entre los tipos de póliza y qué podrían implicar en términos de riesgo y eficiencia operativa?

In [56]:
df.groupby('Policy Type')['Total Claim Amount'].mean()

Policy Type
Corporate Auto    430.312188
Personal Auto     434.828219
Special Auto      440.472770
Name: Total Claim Amount, dtype: float64

Las diferencias que se observan son principalmente que l Special Auto tiene el Total Claim Amount promedio más alto con un valor de 440.47, de manera subsiguiente se encuentra el Personal Auto con un Total Claim Amount promedio de 434.83 y finalmente se encuentra el Corporate Auto tiene el Total Claim Amount promedio más bajo con 430.31.
Esto podrìa implicar en terminos de riesgo y eficiencia corporativa principalmente peor la poliza Scial Auto puesto que son reclamaciones más costosas y genera un mayor riesgo por lo que es necesario ajustar primas y mejorar procesos. En cuanto al Personal Auto presenta un costo de reclamaciones moderado y un riesgo y gestión de reclamaciones intermedios. Y finalmente, las polizas de Corporate Auto son reclamaciones menos costosas, es decir con un menor riesgo y con una oportunidad de precios competitivos.

## Caso de Negocio: Evaluación de la Inversión en Marketing

### Contexto:
Una compañía de seguros ha asignado un presupuesto ficticio de **$5,000,000** para campañas de adquisición. El objetivo es evaluar la rentabilidad de la inversión comparando el gasto de marketing ( 50000000) (CAC) con el Customer Lifetime Value (CLV) obtenido.

### Tareas:
1. Utiliza el CLV promedio obtenido en el Paso 2.
2. Calcula el CAC promedio dividiendo el presupuesto de marketing entre el número total de clientes.
3. Compara el CAC promedio con el CLV promedio y responde:
   - ¿El CAC es inferior al CLV, lo que justificaría la inversión?
   - ¿Qué porcentaje del CLV representa el CAC?

_Reflexión estratégica: Si el CAC es significativamente menor que el CLV, la inversión en marketing es rentable. De lo contrario, se deberá replantear la estrategia de adquisición._

In [57]:
import pandas as pd

# Definir el presupuesto de marketing
marketing_budget = 5000000  # 5 millones de dólares

In [60]:
clv_promedio = df["Customer Lifetime Value"].mean()
print(f"El CLV promedio es {clv_mean} y la desviación estándar es {clv_std}")

El CLV promedio es 8004.940474987081 y la desviación estándar es 6870.967608356924


In [62]:
total_customers = df.shape[0]

In [65]:
cac_promedio = marketing_budget / total_customers

In [66]:
rentabilidad = clv_promedio > cac_promedio
porcentaje_cac_sobre_clv = (cac_promedio / clv_promedio) * 100

In [67]:
print(f"Customer Lifetime Value (CLV) promedio: ${clv_promedio:.2f}")
print(f"Costo de Adquisición de Clientes (CAC) promedio: ${cac_promedio:.2f}")
print(f"El CAC representa el {porcentaje_cac_sobre_clv:.2f}% del CLV.")

Customer Lifetime Value (CLV) promedio: $8004.94
Costo de Adquisición de Clientes (CAC) promedio: $547.41
El CAC representa el 6.84% del CLV.


In [69]:
if rentabilidad:
    print("✅ La inversión en marketing es rentable, ya que el CAC es menor que el CLV.")
else:
    print("❌ La inversión en marketing NO es rentable, ya que el CAC es mayor que el CLV. Se debe replantear la estrategia.")
    print("❌ La inversión en marketing NO es rentable, ya que el CAC es mayor que el CLV. Se debe replantear la estrategia.")

✅ La inversión en marketing es rentable, ya que el CAC es menor que el CLV.


### **Bonus:** Teniendo lo anterior en cual es la proporcion de clientes rentables

## Conclusiones y Recomendaciones

### **Bonus 2:** En no más de un parrafo mencione 3 acciones concretas para mejorar los reusltados de la empresa.


Para maximizar la rentabilidad a largo plazo, las aseguradoras deben adoptar un enfoque integral que equilibre la captación de nuevos clientes, la retención de los existentes y la gestión de riesgos. Esto implica utilizar segmentación precisa y marketing personalizado para atraer a los clientes más valiosos, implementar programas de fidelización y ofrecer un servicio al cliente excepcional para fomentar la lealtad, y optimizar la gestión de riesgos mediante tecnología avanzada y la revisión de pólizas. Al implementar estas estrategias, las aseguradoras pueden reducir costos, aumentar la rentabilidad y optimizar el uso del presupuesto de marketing, asegurando un mejor rendimiento a largo plazo.
